_posts/2018-01-10-CocoaPods-1.4.0.markdown

```diff
@@ -0,0 +1,230 @@
+
+#### Usage
+
+To enable static framework support add the following to your podspec:
+
+```ruby
+s.static_framework = true
+```
+
+Voilà!
+
+### Swift Version DSL
+
+With CocoaPods 1.4.0, you are now able to specify the Swift version your pod officially supports within your podspec via the `swift_version` attribute. Since the release of Xcode 9, multiple Swift versions are now allowed per target to be used. CocoaPods always defaulted to the Swift version your app target was using but as Swift evolves this is no longer sustainable. For example, upgrading your project to Swift 4.0 does not necessarily mean that all pods you consume also successfully compile with Swift 4.0 and therefore you had to wait for a new version of a pod to be published before upgrading to the latest version of Swift.
+
+As a pod author you can now add the following to your podspec:
+
+```ruby
+s.swift_version = '4.0'
+```
+
+This will automatically configure Swift 4.0 for the pod when it's consumed and regardless of the Swift version the app target uses. As an author you should always test and provide the known version of Swift that successfully works with your pod and update your podspec accordingly.
+
+**Note**: The usage of the `.swift-version` file as well as the `--swift-version` parameter during `lint` is now deprecated. You may continue to use it but you will receive a warning to use the `swift_version` attribute instead.
+
+We aim to improve this support in the future by allowing pod authors to specify a range of Swift versions their pod supports instead of a single Swift version, but for the majority this will be sufficient. We would like to remain detached from Apple's Swift roadmap and not to have to perform releases of CocoaPods as new versions of Swift and Xcode are introduced.
+
+### Transition to `https`
+
+With this release, publishing a pod using an unencrypted `http` source is now deprecated and it is *highly* recommended to begin the process of hosting your pod using an `https` source. With this release, a warning will appear if you attempt to publish a pod in the master repo that's not using an `https` source. In a future release this will become more strict and you will be unable to publish a pod without an `https` source.
+
+Additionally, during installation consumers of pods with unencrypted `http` source will also receive a warning notifying them that the source in which the pod is hosted may be unsafe. An option to disable this warning `bla_bla_blah` is available for you to disable this warning but its usage is highly discouraged.
+
+### Development Pod Files
+
+It is often very useful when you are developing locally on a pod to have all of its associated files (such as documentation files, its podspec and license files) to be easily available and visible so you can view and edit them. In 1.4.0, all of these files are now automatically added to the generated `.xcodeproj`! The following files will be automatically added to the workspace:
+
```

Comment:
I would add a note specifying what is meant by developing locally - is there a link to use for a reference for this feature? Or just specify that we mean pods installed with the `:path` option

Reply:
added a quick parenthesis